Makefile

```diff
@@ -194,6 +194,7 @@ modules: ## Runs go mod to ensure modules are up to date.
 	go mod tidy
 	cd $(TOOLS_DIR); go mod tidy
 	cd $(E2E_FRAMEWORK_DIR); go mod tidy
+	cd test/infrastructure/docker; go mod tidy
```

Comment:
For followup, we should probably also consider doing similar against the test/infrastructure/docker/hack/tools directory also.